iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -68,6 +70,8 @@ static AmqpTransportHandler()
             // Create a timer to remove any expired messages.
             _twinTimeoutTimer = new Timer(CheckTimeout);
 
+            _connectionCredentials = context.IotHubConnectionCredentials;
```

Comment:
small nit: I'd recommend saving this local variable above the _amqp initialization and passing it in L60. That way we'll be using the same local variable everywhere.